message/validation/consensus_validation.go

```diff
@@ -147,6 +147,24 @@ func (mv *messageValidator) validateConsensusMessage(
 	return consensusDescriptor, msgSlot, nil
 }
 
+func (mv *messageValidator) allowedRoundRange(estimatedRound specqbft.Round, maxRound specqbft.Round) (specqbft.Round, specqbft.Round) {
+	var lowestAllowed, highestAllowed specqbft.Round
+
+	if lowestAllowed > allowedRoundsInPast {
```

Comment:
this is here in case `allowedRoundsInPast` is changed? since `lowestAllowed` will always be 0

Reply:
this must be `if estimatedRound > allowedRoundsInPast {`